qualtrics_link/views.py

```diff
@@ -199,7 +199,7 @@ def internal(request):
                                                  person_details.last_name,
                                                  person_details.email,
                                                  person_details.role,
-                                                 person_details.division)
+                                                 util.DIVISION_MAPPING[person_details.division])
```

Comment:
This will cause a failure/exception is if person_details.division is missing from the mapping. We should code for an unexpected value to be there, so Qualtrics will continue to put  it into the Other bucket. Right now an exception will be thrown

Reply:
There is already handling in place for that in the creation of the PersonDetails object. This is created using the get_person_details method in util. If for some reason the persons division within the DB does not match any of the divisions listed in the util file, it defaults to 'Other'. DIVISION_MAPPING would then be mapping 'Other' to the Qualtrics ID of 'DV_1zu8x43ZIyqzWlu'

Reply:
ok, I see the default in  PersonDetails. I  tried sending an unknown division in the management command and it failed with an exception.

Reply:
Testing an unknown value at this specific point is not the correct way to test that seeing that the default would have been set at that point. At lines 374 and/or 381 or util would be the place to look.

Reply:
That's why I pointed out where I was seeing the exception :-) . The test command was an easy place to test but it is not using the underlying methods the same as the actual code. This project doesn't have test cases, but this is an apt case for why we need it.

I would also second Colin's point about escaping  the values being passed.